packages/@lwc/integration-karma/test/component/LightningElement/index.spec.js

```diff
@@ -16,6 +17,15 @@ it('should throw when trying to invoke the constructor manually', () => {
     }).toThrowError(ReferenceError);
 });
 
+// TODO [#2970]: component constructor cannot be new-ed
+it('throws when trying to `new` a compiled subclass of LightningElement', () => {
+    const func = () => {
+        new Component();
+    };
+    expect(func).toThrowError(ReferenceError);
+    expect(func).toThrowError(/Illegal constructor/);
+});
+
```

Comment:
There's a test above this one that looks similar, but it's not quite the same because this component is compiled rather than created manually (`class Test extends LightningElement {}`).

Maybe it's not really different in practice, but I wanted to cover the bases here.